### PR TITLE
More native contract RPC wrappers

### PR DESCRIPTION
**Pull request by roman-khimov**
Another batch of wrappers.

### CODE REVIEW

File: pkg/rpcclient/management/management.go [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+}
+
+// Hash stores the hash of the native ContractManagement contract.
+var Hash = state.CreateNativeContractHash(nativenames.Management)
+
+// Event is the event emitted on contract deployment/update/destroy.
+// Even though these events are different they all have the same field inside.
+type Event struct {
+	Hash util.Uint160
+}
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/management/management.go:65]: We'll have some API to extract these structures from notifications eventually, that's when the names will become more useful.

File: pkg/rpcclient/management/management.go [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+	MakeCall(contract util.Uint160, method string, params ...interface{}) (*transaction.Transaction, error)
+	MakeRun(script []byte) (*transaction.Transaction, error)
+	MakeUnsignedCall(contract util.Uint160, method string, attrs []transaction.Attribute, params ...interface{}) (*transaction.Transaction, error)
+	MakeUnsignedRun(script []byte, attrs []transaction.Attribute) (*transaction.Transaction, error)
+	SendCall(contract util.Uint160, method string, params ...interface{}) (util.Uint256, uint32, error)
+	SendRun(script []byte) (util.Uint256, uint32, error)
+}
+
+// ContractReader provides an interface to call read-only PolicyContract
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/management/management.go]: Yeah, we need some autogeneration here. Fixed.

File: pkg/rpcclient/management/management.go [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+// contract's methods.
+type ContractReader struct {
+	invoker Invoker
+}
+
+// Contract represents a PolicyContract contract client that can be used to
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/management/management.go]: Fixed.

File: pkg/rpcclient/management/management.go [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+// invoke all of its methods except 'update' and 'destroy' because they can be
+// called successfully only from the contract itself (that is doing an update
+// of self-destruction).
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/management/management.go]: Fixed.

File: pkg/rpcclient/oracle/oracle.go [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+// ContractReader provides an interface to call read-only OracleContract
+// contract's methods. "verify" method is not exposed since it's very specific
+// and can't be executed successfully outside of the proper oracle response
+// transaction.
+type ContractReader struct {
+	invoker Invoker
+}
+
+// Contract represents the OracleContract contract client that can be used to
+// invoke its "setPrice" method. Other methods are useless for direct calls,
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/oracle/oracle.go:48]: OK.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**